src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEReader.cs

```diff
@@ -675,6 +677,9 @@ internal static void ValidateEmbeddedPortablePdbVersion(DebugDirectoryEntry entr
         // internal for testing
         internal static unsafe ImmutableArray<byte> DecodeEmbeddedPortablePdbDebugDirectoryData(AbstractMemoryBlock block)
         {
+#if CORERT
+			throw new PlatformNotSupportedException();
```

Comment:
We'll simply not successfully find these pdbs. (done through partial method trickery)